src/app/core/models/templates/sections/tester-response-section.model.ts

```diff
@@ -36,12 +37,14 @@ export class TesterResponseSection extends Section {
             this.teamChosenType = this.parseTeamChosenType(description);
           }
 
+          this.testerDisagree = this.parseCheckboxValue(disagreeCheckbox);
```

Comment:
There's a bug with this line - if the team disagrees on two or more parts (e.g. issue type and severity), this will just be set to the disagree checkbox of the last part. E.g. if I disagree on issue type but agree on severity, `this.testerDisagree` will be false, which is not the intended behaviour. See the gif below:
![wrong agree disagree](https://user-images.githubusercontent.com/68138671/155845013-df50d0fa-0950-4ea1-acd5-56d384644cc3.gif)

Reply:
Thank you, will change it to Response = Disagree on disagree with any part (response, severity, or type).